mlir/lib/Dialect/Vector/Transforms/VectorLinearize.cpp

```diff
@@ -103,6 +105,234 @@ struct LinearizeVectorizable final
     return success();
   }
 
+private:
+  unsigned targetVectorBitWidth;
+};
+
+struct LinearizeVectorExtractStridedSlice final
```

Comment:
Please add comments summarising this transformation. Would be good to have that for other patterns too.

Reply:
Added some comments describing what the rewrite pattern does.

Reply:
Thanks for pointing this out. I tried my best to add some example based comments for each pattern.